src/openrct2/libopenrct2.vcxproj

```diff
@@ -50,19 +50,720 @@
     <None Include="data\shaders\**\*.vert" />
     <None Include="data\shaders\**\*.frag" />
   </ItemGroup>
-  <ItemGroup>
-    <ClCompile Include="**\*.cpp" />
-  </ItemGroup>
   <ItemGroup>
     <ClInclude Include="..\..\resources\resource.h" />
-    <ClInclude Include="**\*.h" />
-    <ClInclude Include="**\*.hpp" />
```

Comment:
This change is not related to the rest.
(Also, do we really need to do this?)

Reply:
I will exclude this change. But since the VS update it will be something that will keep ending up in PRs. Maybe I can disable tracking for the file...

Reply:
We will have to do this unfortunately. If we turn off tracking it's just going to be annoying when they eventually disable the feature completely.